src/App.tsx

```diff
@@ -4,54 +4,60 @@ import { HomeOutlined } from '@ant-design/icons';
 import { CampaignDetails } from './CampaignsAdmin/edit/CampaignDetails';
+import { Signup } from 'Auth/Signup';
+import { Login } from 'Auth/Login';
 import 'antd/dist/antd.css';
 import './index.css';
 import { Map } from './Maps/Map';
 import { Report } from './CampaignsAdmin/report/Report';
-
+import { AuthProvider } from 'contexts/Auth';
+import { PrivateRoute } from 'Auth/PrivateRoute';
 const { Header, Content, Footer } = Layout;
 function App() {
   return (
     <Router>
-      <Header>
-        <Menu theme="dark" mode="horizontal">
-          <Menu.Item key="home">Home</Menu.Item>
-          <Menu.Item key="campaigns">
-            <Link to="/CampaignsAdmin">Campaigns</Link>
-          </Menu.Item>
-          <Menu.Item key="locations">
-            <Link to="/LocationsAdmin">Locations</Link>
-          </Menu.Item>
-        </Menu>
-      </Header>
-
-      <Content style={{ padding: '0 50px', margin: '5px 0px', textAlign: 'right' }}>
-        <Breadcrumb style={{ margin: '5px 0' }}>
-          <Breadcrumb.Item>
-            <HomeOutlined />
-          </Breadcrumb.Item>
-          <Breadcrumb.Item>Campaigns</Breadcrumb.Item>
-        </Breadcrumb>
-      </Content>
-      <div style={{ padding: '0px 0px 20px 0px' }} className="site-layout-content">
-        <Layout style={{ height: 'auto' }} className="layout">
-          <div style={{ margin: '0 2%' }}>
-            <Switch>
-              <Route path="/LocationsAdmin" exact component={LocationsAdmin} />
-              <Route path="/CampaignsAdmin" exact component={CampaignAdmin} />
-              <Route path="/CampaignsAdmin/edit/:id"  component={CampaignDetails} />
-              <Route path="/CampaignsAdmin/report/:id" exact component={Report} />
-              <Route path="/MapTest" component={Map} />
-            </Switch>
-          </div>
-
-          <Footer
-            style={{ textAlign: 'center', backgroundColor: '#173057', color: 'white', fontSize: '1em' }}
-          >
-            What do we say to the Footer God? NOT TODAY
-          </Footer>
-        </Layout>
-      </div>
+      <AuthProvider>
+        <Header>
+          <Menu theme="dark" mode="horizontal">
+            <Menu.Item key="home">HOME</Menu.Item>
+            <Menu.Item key="campaigns">
+              <Link to="/CampaignsAdmin">Campaigns</Link>
+            </Menu.Item>
+            <Menu.Item key="locations">
+              <Link to="/LocationsAdmin">Locations</Link>
+            </Menu.Item>
+          </Menu>
+        </Header>
+        <Content style={{ padding: '0 50px', margin: '5px 0px', textAlign: 'right' }}>
+          <Breadcrumb style={{ margin: '5px 0' }}>
+            <Breadcrumb.Item>
+              <HomeOutlined />
+            </Breadcrumb.Item>
+            <Breadcrumb.Item>Campaigns</Breadcrumb.Item>
+          </Breadcrumb>
+        </Content>
+        <div style={{ padding: '0px 0px 20px 0px' }} className="site-layout-content">
```

Comment:
Create a new issue to remove all inline style. If we're using Scss let's stick to assigning classnames instead

Reply:
Did you create this issue in Linear?